haskell/test/Test/Ligo/BaseDAO/Management/TransferOwnership.hs

```diff
@@ -72,6 +72,7 @@ changeToPendingAdmin withOriginatedFn initialStorage =
       withSender owner $ call baseDao (Call @"Transfer_ownership")
         (#newOwner .! wallet1)
       withSender wallet1 $ call baseDao (Call @"Accept_ownership") ()
+      -- TODO@SRAS Add test to check new owner is set
```

Comment:
Maybe we should create a new issue for this?

Reply:
Ah, I missed to fill this. Have included the check now.